src/DssSpell.sol

```diff
@@ -52,82 +59,166 @@ contract DssSpellAction is DssAction {
         return true;
```

Comment:
Shouldn't this be set to false now that the Coinbase Custody onboarding was dropped?

Reply:
Actually, the [sheet](https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit#gid=50030571&range=C3) says it should be set to `false`.

Reply:
I think we agreed on leave office hours enable as we do some payments and streams

Reply:
ahh, ok we have it disabled in exec sheets